### DISCUSSION TITLE
Directus 9.21 - Cannot access shared types anymore

### DISCUSSION THREAD

**Discussion by gkielwasser** [directus/directus#16601] [Archive]
Hi,

Starting from @directus/shared 9.21.0 I cannot access anymore the types. Is it on purpose not to expose them anymore ?

Mainly I was using this way to type hook register function:

```
import { HookConfig } from "@directus/shared/types";
const registerHook: HookConfig = ({ filter, action, init }, {...}) => {}
```

Thanks for help

**Comment by paescuj** [comment]
Thanks for reporting! Fixed this in #16606.

**Reply by gkielwasser** [reply]
Thanks !
